importer/views.py

```diff
@@ -0,0 +1,43 @@
+"""
+Views for the importer app.
+"""
+
+from django.shortcuts import render
+from django.contrib.auth.models import User
+
+from learningresources.models import Repository, Course, LearningResource
+from .forms import UploadForm
+
+
+def status(request):
+    """
+    Show what's currently in the system.
+    """
+    counts = {
+        "repos": Repository.objects.count(),
+        "courses": Course.objects.count(),
+        "loxes": LearningResource.objects.count(),
+    }
+    return render(
+        request,
+        "status.html",
+        counts,
+    )
+
+
+def upload(request):
+    """
+    Upload a OLX archive.
+    """
+    form = UploadForm()
+    if request.user.is_anonymous():
+        request.user, _ = User.objects.get_or_create(username="dirty_hack")
```

Comment:
Master is running live on Heroku: https://lore-ci.herokuapp.com.  Yeah, you can either keep the TODO and disable that pylint check, or put the comment in without it as is.  Heroku has a 30MiB file limit, which will of course be a problem pretty quickly, so I don't think we can run prod on it without switching to using something like S3 client side uploads (which we probably want to do anyway).